pkg/gui/views.go

```diff
@@ -205,6 +207,12 @@ func (gui *Gui) createAllViews() error {
 	gui.Views.CommitMessage.Editable = true
 	gui.Views.CommitMessage.Editor = gocui.EditorFunc(gui.commitMessageEditor)
```

Comment:
gui.commitMessageEditor has resizing logic that we don't need if the commit message view doesn't need to resize as we type, so we can remove that resizing logic from it